### ISSUE TITLE
Bug: Disabling rule with VSCode context menu broken when there is description

### ISSUE THREAD

**Issue by ilyub** [eslint/eslint#16157]
### Environment

Node version: 18.4.0
npm version: 8.12.1
Local ESLint version: 8.20.0
Global ESLint version: 
Operating System: Win 10

### What parser are you using?

vue-eslint-parser

### What did you do?

1. Disable some rule in VSCode:
```ts
// eslint-disable-next-line rule1
```
![image](https://user-images.githubusercontent.com/70687432/180714886-bcc5b5c4-90fb-41c9-83a6-fea94df4fadc.png)

2. Add comment
```
// eslint-disable-next-line rule1 -- This rule is disabled because ...
```

3. Disable another rule:
```
// eslint-disable-next-line rule1 -- This rule is disabled because ..., rule2
```

### What did you expect to happen?

```
// eslint-disable-next-line rule1, rule2 -- This rule is disabled because ...
```

### What actually happened?

```
// eslint-disable-next-line rule1 -- This rule is disabled because ..., rule2
```

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by mdjermanovic** [comment]
Hi @ilyub, thanks for the issue! 

> ![image](https://user-images.githubusercontent.com/70687432/180714886-bcc5b5c4-90fb-41c9-83a6-fea94df4fadc.png)

ESLint doesn't provide an API for adding eslint-disable comments. This feature is implemented in the vscode-eslint extension. If it doesn't work as expected, you can open an issue in this repository:

https://github.com/microsoft/vscode-eslint

**Comment by ilyub** [comment]
Thx. Submitted issue as you suggested:
https://github.com/microsoft/vscode-eslint/issues/1505
